packages/modules/notification/src/migrations/Migration20240627153332.ts

```diff
@@ -0,0 +1,24 @@
+import { Migration } from "@mikro-orm/migrations"
+
+export class Migration20240627153332 extends Migration {
+  async up(): Promise<void> {
+    this.addSql(
+      'alter table if exists "notification_provider" add column if not exists "created_at" timestamptz not null default now(), add column "updated_at" timestamptz not null default now(), add column "deleted_at" timestamptz null;'
+    )
+    this.addSql(
+      'alter table if exists "notification_provider" alter column "channels" type text[] using ("channels"::text[]);'
+    )
+    this.addSql(
+      'alter table if exists "notification_provider" alter column "channels" set default \'{}\';'
+    )
+
+    this.addSql(
+      'alter table if exists "notification" add column if not exists "updated_at" timestamptz not null default now(), add column "deleted_at" timestamptz null;'
+    )
+    this.addSql('drop index if exists "IDX_notification_idempotency_key";')
+    this.addSql('drop index if exists "IDX_notification_provider_id";')
```

Comment:
@riqwan I would expect this to not be dropped, since it's a `belongsTo` relationship. Did I maybe misunderstand how it works?

Reply:
It shouldn't be removed. Let me take a look

Reply:
good catch - https://github.com/medusajs/medusa/pull/7858/files